cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -269,6 +269,21 @@ spec:
         #  value: "9080"
         - name: ROOK_HOSTPATH_REQUIRES_PRIVILEGED
           value: "true"
+
+        # Time to wait until the node controller move Rook pods to other nodes
+        # after detecting a unreachable node.
+        # Pods affected by this setting are:
+        # mgr, rbd, mds, rgw, PVC based mons and osds, and ceph toolbox
+        # The value used in this variable replaces the default value of 300 secs
+        # added automatically by k8 as Toleration for
+        # <node.kubernetes.io/unreachable>
+        # The total amount of time to reschedule Rook pods in healthy nodes
+        # before detecting a <not ready node> condition will be the sum of:
+        #  --> node-monitor-grace-period: 40 seconds (node controller flag)
+        #  --> UNREACHABLE_NODE_TOLERATION_SECONDS: 5 seconds
+        - name: ROOK_UNREACHABLE_NODE_TOLERATION_LAPSE
+          value: "7s"
```

Comment:
Why is this set to 7 here? Why does this require a unit? Would this be more usable as `ROOK_UNREACHABLE_NODE_TOLERATION_SECONDS` and accept only integer values? This seems less likely to fail compared to a user either failing to input a unit or inputting an invalid unit.

Reply:
Is 7 because i forget to return it to 5 after some test ... :-) .. thanks for the finding!!!
About the way to specify time in the new setting:
I only has followed the same criteria used previously in another settings where the value stored is time, example:

```
        - name: ROOK_MON_HEALTHCHECK_INTERVAL
          value: "45s"
        - name: ROOK_MON_OUT_TIMEOUT
          value: "600s"
        - name: ROOK_DISCOVER_DEVICES_INTERVAL
          value: "60m"

```
I think that is good to follow the preexisting rules/criteria but i see your point. So we can discuss this question in the huddle and see if it is worth to change all this settings to use only integer values and include in the name of the setting the time unit used.